### DISCUSSION TITLE
Including files in the dock by default

### DISCUSSION THREAD

**Discussion by Suzie97** [elementary/files#1727] [Ideas]
Whenever I do a clean installation of elementary, I look for the files app on the dock. But soon I realize that it is not there.

I've seen other users talking about this too.

Maybe it might be a good idea to include files in the dock by default since it is a pretty essential and frequently used tool.

**Comment by jeremypw** [comment]
What are the operations that *all or most* users (especially non-technical ones) need to and that cannot be done except by launching the filemanager? I think the aim is to reduce that number to zero although I suspect it is not there yet.  Examples I can think of might be mounting devices and connecting to servers.

**Comment by tintou** [comment]
Here is the answer: https://github.com/elementary/default-settings/issues/191

**Comment by jeremypw** [comment]
Yes, if we want to expose functions like mounting drives and network servers then maybe a new dedicated app or indicator is more appropriate.
